### ISSUE TITLE
Lab Report 2 Feedback

### ISSUE THREAD

**Issue by joy002** [PiaoX/cse15l-lab-reports#2]
**Bug/Input 1**
- [x] Commit screenshot for bug/input 1
- [x] Link to test file for bug/input 1
   - Need a link to test file for this bug/input
- [x] Symptom shown as command + output for bug/input 1
- [x] Description of relationship between bug, symptom, and input for bug/input 1

**Bug/Input 2**
- [x] Commit screenshot for bug/input 2
- [x] Link to test file for bug/input 2
   - Need a link to test file for this bug/input
- [x] Symptom shown as command + output for bug/input 2
- [x] Description of relationship between bug, symptom, and input for bug/input 2

**Bug/Input 3**
- [x] Commit screenshot for bug/input 3
- [x] Link to test file for bug/input 3
   - Need a link to test file for this bug/input
- [x] Symptom shown as command + output for bug/input 3
- [x] Description of relationship between bug, symptom, and input for bug/input 3

Additional comments from your grader:
Good Job :)

Feel free to comment below if you have any questions about the rubric/feedback.

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by joy002** [comment]
Hi, you still didn't add the link, please add the links to your test file and submit regrade again.

**Comment by PiaoX** [comment]
Hi,

It doesn’t allow me to submit regrade again, does it only allow regrade
once?

Best,

Piao

**Comment by joy002** [comment]
No, you can regrade more than once. Wait a few minutes and try again. Tell me if it still doesn't work.

**Comment by PiaoX** [comment]
It’s still doesn’t allow me to submit a regrade, I’ll wait until tomorrow
and if it doesn't work, I'll let you know.

Thanks for your help!

Best,

Piao

**Comment by joy002** [comment]
Hi, sorry but you need to make the link clickable and not just pasting the address.

**Comment by joy002** [comment]
Nice job:) I have updated your grade. Feel free to ask me question if there's any. Please close this issue if there is no question.
